lib/screens/study_nurse_screen/configuration_screen/configuration_pages.dart

```diff
@@ -0,0 +1,135 @@
+}
+
+class _ConfigurationPages extends State<ConfigurationPages> {
+  PageController controller = PageController();
+
+  final List<Widget> _list = <Widget>[
+    const ConfigurationForm(),
+    const ConfigurationVitalValues(),
+    const ConfigurationActivityAndRest(),
+    const ConfigurationBodyAndMind(),
+    const ConfigurationMedicationAndTherapy(),
+    const ConfigurationSummary()
+  ];
+
+  int _currentPage = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    final GlobalTheme theme = Theme.of(context).extension<GlobalTheme>()!;
+
+    return Scaffold(
+      backgroundColor: Colors.white,
+      appBar: AppBar(
+        title: Text(AppLocalizations.of(context)!.configuration),
+        backgroundColor: theme.darkGreen3,
+      ),
+      body: Form(
+        key: formKeyConfiguration,
+        child: PageView(
+          scrollDirection: Axis.vertical,
+          controller: controller,
+          onPageChanged: (int num) {
+            setState(() {
+              _currentPage = num;
+            });
+          },
+          children: _list,
+        ),
+      ),
+      bottomNavigationBar: Row(
+        children: <Widget>[
+          Expanded(
+            child: PicosInkWellButton(
+              text: AppLocalizations.of(context)!.back,
+              onTap: () {
+                if (_currentPage == 0) {
+                  Navigator.of(context).pop();
+                } else {
+                  controller.jumpToPage(_currentPage - 1);
+                }
+              },
+              buttonColor1: theme.grey3,
+              buttonColor2: theme.grey1,
+            ),
+          ),
+          Expanded(
+            child: PicosInkWellButton(
+              text: AppLocalizations.of(context)!.proceed,
+              onTap: () async {
+                if (_currentPage == _list.length - 1) {
+                  formKeyConfiguration.currentState!.save();
+                  bool backendCreatePatient = await Backend.createPatient();
+                  if (backendCreatePatient) {
+                    if (!mounted) return;
+                    Navigator.of(context)
+                        .pushNamed('/configuration-finish-screen');
+                  } else {
+                    log('Fehler beim Anlegen des Patienten!');
+                  }
+                } else {
+                  if (_currentPage == 0) {
+                    if (formKeyConfiguration.currentState!.validate()) {
+                      ScaffoldMessenger.of(context).showSnackBar(
+                        SnackBar(
+                          content: Text(
+                            AppLocalizations.of(context)!.submitData,
+                          ),
+                        ),
+                      );
+                      controller.jumpToPage(_currentPage + 1);
+                    }
+                  } else {
+                    controller.jumpToPage(_currentPage + 1);
+                  }
```

Comment:
That is a lot of unnecessary condition nesting, even with code duplication.

Reply:
This should be fixed now.